recipes/libdispatch/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class LibDispatchConan(ConanFile):
+    name = "libdispatch"
+    homepage = "https://github.com/apple/swift-corelibs-libdispatch"
+    description = "Grand Central Dispatch (GCD or libdispatch) provides comprehensive support for concurrent code execution on multicore hardware.."
+    topics = ("conan", "libdispatch", "apple", "GCD", "concurrency")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _cmake = None
+
+    def validate(self):
+        if self.settings.build_type != "Release":
+            raise ConanInvalidConfiguration("Just Release builds allowed.")
+
+        if self.settings.compiler != "clang":
+            raise ConanInvalidConfiguration("Clang compiler is required.")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = f'swift-corelibs-{self.name}-swift-{self.version}-RELEASE'
```

Comment:
```suggestion
        extracted_dir = "swift-corelibs-{}-swift-{}-RELEASE".format(self.name, self.version)
```

Python format string is not supported by Python 2.7